components/raftstore/src/store/fsm/apply.rs

```diff
@@ -1054,7 +971,6 @@ where
         }
 
         apply_ctx.cbs.last_mut().unwrap().push(cmd_cb, cmd);
```

Comment:
The code here depends on the fact that the last `cb` in `cbs` belongs to this region. But it may be not true if meeting context switch.